### PR TITLE
[T7][F12-B3]

### PR DESCRIPTION
**Pull request by MaggieMeow**
Ready for review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,17 @@
-# Address Book (Level 4)
-
-[![Build Status](https://travis-ci.org/se-edu/addressbook-level4.svg?branch=master)](https://travis-ci.org/se-edu/addressbook-level4)
+# Task Manager
+[![Build Status](https://travis-ci.org/CS2103JAN2017-F12-B3/main.svg?branch=master)](https://travis-ci.org/CS2103JAN2017-F12-B3/main.svg?branch=master)
```
Comment (okkhoy) [inline README.md]: - similar to this, you need to update the other badges (coverall, codacy etc.,) to reflect to your project.
-  in the above, the title should reflect your project, not addressbook anymore.

File: README.md [hunk 2]
```diff
@@ -1,22 +1,17 @@
...
 [![Build status](https://ci.appveyor.com/api/projects/status/3boko2x2vr5cc3w2?svg=true)](https://ci.appveyor.com/project/damithc/addressbook-level4)
 [![Coverage Status](https://coveralls.io/repos/github/se-edu/addressbook-level4/badge.svg?branch=master)](https://coveralls.io/github/se-edu/addressbook-level4?branch=master)
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/fc0b7775cf7f4fdeaf08776f3d8e364a)](https://www.codacy.com/app/damith/addressbook-level4?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=se-edu/addressbook-level4&amp;utm_campaign=Badge_Grade)
 
-<img src="docs/images/Ui.png" width="600"><br>
+<img src="docs/images/uiv0.0.png" width="600"><br>
```
Comment (okkhoy) [inline README.md]: good to see a mock up of your product :+1:

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -120,15 +120,15 @@ _Figure 2.1.2 : Class Diagram of the Logic Component_
 The _Sequence Diagram_ below shows how the components interact for the scenario where the user issues the
 command `delete 1`.
 
-<img src="images\SDforDeletePerson.png" width="800"><br>
+<img src="images\SDforDeleteTask.png" width="800"><br>
```
Comment (okkhoy) [inline docs/DeveloperGuide.md:123]: in the sequence diagram(s), there is some minor issue with typesetting, the command words are wrapping around arrows. please try to fix them.
I wont make the same comment again for the others, too lazy to type :-p

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -139,19 +139,19 @@ The sections below give more details of each component.
...
 ### 2.2. UI component
 
-Author: Alice Bee
+Author: Shen Qi
 
 <img src="images/UiClassDiagram.png" width="800"><br>
 _Figure 2.2.1 : Structure of the UI Component_
 
-**API** : [`Ui.java`](../src/main/java/seedu/address/ui/Ui.java)
+**API** : [`Ui.java`](../src/main/java/seedu/taskmanager/ui/Ui.java)
```
Comment (okkhoy) [inline docs/DeveloperGuide.md:147]: this still refers to person and addressbook related stuff, please fix it.

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -162,57 +162,57 @@ The `UI` component,
...
 ### 2.5. Storage component
 
-Author: Darius Foong
+Author: Aditya
 
 <img src="images/StorageClassDiagram.png" width="800"><br>
 _Figure 2.5.1 : Structure of the Storage Component_
 
-**API** : [`Storage.java`](../src/main/java/seedu/address/storage/Storage.java)
+**API** : [`Storage.java`](../src/main/java/seedu/taskmanager/storage/Storage.java)
```
Comment (okkhoy) [inline docs/DeveloperGuide.md:206]: same comment as above, please remove references to address book in the image.

File: docs/DeveloperGuide.md [hunk 6]
```diff
@@ -377,9 +402,262 @@ Use case ends.
...
+> 3a1. TaskManager shows an error message <br>
   Use case resumes at step 2
 
+#### Use case: UC02 - Add new task with a task description
+
+**MSS**
+
+1. User enter the new task to be added with all the details including date/time, tags, and description
+2. TaskManager accepts the new task with a confirmation
+Use case ends.
```
Comment (okkhoy) [inline docs/DeveloperGuide.md:414]: some issue with new line here. this appears in the same line as step 2.

File: docs/DeveloperGuide.md [hunk 7]
```diff
@@ -377,9 +402,262 @@ Use case ends.
...
+**Extensions**
+
+1a. The input is not valid
+
+> The TaskManager ask for the correct input <br>
+  Use case resumes at step 1
+
+2a. User requests to undo the action
+
+> 2a1. TaskManager shows that undo is successful <br>
```
Comment (okkhoy) [inline docs/DeveloperGuide.md:425]: i am not very sure if undo should be captured as an extension here?

File: docs/DeveloperGuide.md [hunk 8]
```diff
@@ -377,9 +402,262 @@ Use case ends.
...
+  Use case ends
+
+#### Use case: UC03 - Edit task description
```
Comment (okkhoy) [inline docs/DeveloperGuide.md]: while this is a very well written use case (:+1:), it gives me an impression that the edit command is a multi step command.
please remember that the commands need to be "one shot" commands.

File: docs/DeveloperGuide.md [hunk 9]
```diff
@@ -377,9 +402,262 @@ Use case ends.
...
+  Use case ends
+
+#### Use case: UC06 - Find a task with a keyword
+
+**MSS**
+
+1. User requests to find task by keyword(s)
+2. TaskManager shows a list of tasks containing given keyword
+3. User selects a task to view in detail.
+4. User can request to make an edit of the task (UC03, UC04, UC05, or UC09)
```
Comment (okkhoy) [inline docs/DeveloperGuide.md]: this step doesn't belong to this use case. hence can be removed.
instead, you need to mention that after user selects the task, the system will show details of the selected task or something like that.

File: docs/DeveloperGuide.md [hunk 10]
```diff
@@ -377,9 +402,262 @@ Use case ends.
...
+  Use case resumes at step 2
+
+#### Use case: UC07 - Find a task by a tag
+
+**MSS**
+
+1. User requests to find task by a tag
+2. TaskManager shows a list of tasks containing given tag
+3. User selects a task to view in detail.
+4. User can request to make an edit of the task (UC03, UC04, UC05, or UC09)
```
Comment (okkhoy) [inline docs/DeveloperGuide.md]: same comment as above.

File: docs/DeveloperGuide.md [hunk 11]
```diff
@@ -377,9 +402,262 @@ Use case ends.
...
+  Use case resumes at step 2
+
+#### Use case: UC08 - Find a task to do today
+
+**MSS**
+
+1. User requests to find task to do today
+2. TaskManager shows a list of tasks with deadlines/reminder scheduled today
+3. User selects a task to view in detail
+4. User can request to make a further action with of the task (UC03, UC04, UC05, or UC09)
```
Comment (okkhoy) [inline docs/DeveloperGuide.md]: ... and again

File: docs/DeveloperGuide.md [hunk 12]
```diff
@@ -377,9 +402,262 @@ Use case ends.
...
+  Use case resumes at step 2
+
+4a. User requests to undo the action
+
+> 4a1. TaskManager shows that undo is successful <br>
+  Use case ends
+
 {More to be added}
 
 ## Appendix C : Non Functional Requirements
```
Comment (okkhoy) [inline docs/DeveloperGuide.md:570]: your NFR still refers to persons :-P please fix it.

File: docs/UserGuide.md [hunk 13]
```diff
@@ -16,171 +16,140 @@ By : `Team SE-EDU`  &nbsp;&nbsp;&nbsp;&nbsp; Since: `Jun 2016`  &nbsp;&nbsp;&nbs
...
 > Help is also shown if you enter an incorrect command e.g. `abcd`
 
-### 2.2. Adding a person: `add`
+A command guide will be shown in the window
+
+### 2.2. Adding a task: `add`
 
-Adds a person to the address book<br>
-Format: `add NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...`
+Adds a task to the task manager<br>
```
Comment (okkhoy) [inline docs/UserGuide.md:46]: how about adding tasks with both start and end times?

File: docs/UserGuide.md [hunk 14]
```diff
@@ -16,171 +16,140 @@ By : `Team SE-EDU`  &nbsp;&nbsp;&nbsp;&nbsp; Since: `Jun 2016`  &nbsp;&nbsp;&nbs
...
 
-* `edit 2 Betsy Crower t/`<br>
-  Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
+### 2.5. Editing a task : `edit`
 
-### 2.5. Finding all persons containing any keyword in their name: `find`
+Edits an existing task in the task manager.<br>
+The tasks can be edited through the index from a list
+Format: `edit INDEX`
+Follow subsequent instructions to edit the respective items such as timing, tag, etc.
```
Comment (okkhoy) [inline docs/UserGuide.md]: you have spacing issues here. may need to add a newline character explicitly.

Comment (okkhoy) [reply docs/UserGuide.md]: i still get an impression that edit is a multi part command? if that is the case, please fix it in the design itself! the commands should be "one shot"

File: docs/UserGuide.md [hunk 15]
```diff
@@ -16,171 +16,140 @@ By : `Team SE-EDU`  &nbsp;&nbsp;&nbsp;&nbsp; Since: `Jun 2016`  &nbsp;&nbsp;&nbs
...
 ### 2.8. Clearing all entries : `clear`
 
-Clears all entries from the address book.<br>
+Clears all entries from the task manager.<br>
 Format: `clear`
 
-### 2.9. Exiting the program : `exit`
+### 2.9. Viewing history of task : `history`
+
+A list of history of tasks will be shown in the column sorted in reverse-chronological order
```
Comment (okkhoy) [inline docs/UserGuide.md]: apart from sorting, what is the difference between this and list command?

File: docs/UserGuide.md [hunk 16]
```diff
@@ -16,171 +16,140 @@ By : `Team SE-EDU`  &nbsp;&nbsp;&nbsp;&nbsp; Since: `Jun 2016`  &nbsp;&nbsp;&nbs
...
 **Q**: How do I transfer my data to another Computer?<br>
 **A**: Install the app in the other computer and overwrite the empty data file it creates with
-       the file that contains the data of your previous Address Book folder.
+       the file that contains the data of your previous Task Manager folder.
 
 ## 4. Command Summary
 
-* **Add**  `add NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...` <br>
-  e.g. `add James Ho p/22224444 e/jamesho@gmail.com a/123, Clementi Rd, 1234665 t/friend t/colleague`
+* **Help** : `help` <br>
```
Comment (okkhoy) [inline docs/UserGuide.md]: can consider providing command summary as a table instead.

**Review by okkhoy — COMMENTED** [review]
This is a commendable effort in updating the user guide and developer guide. Kudos!! 

- you have a good set of product surveys :+1:  
- i have made some comments on the developer guide diagrams. though it is not necessary for this submission, i can see you have worked through some of it. i appreciate it :+1:  but you invited trouble 'cos now i am pointing out mistakes in the diagrams :-P 

some concerns:
- please take a look at the product requirements and scope and see if you have captured everything in that in your user stories/user guide. 
- you seem to have missed some basic variations of commands, please take a look a the details.
- some parts of your user guide are terse. try and fix them.

- please ensure readme file reflects your project and not addressbook.
- please update the about us page to reflect your team.

**Comment by okkhoy** [comment]
@MaggieMeow @gabrielsim1993 @tangshenqi @ahendraatmaja 
Some comments added. Please ack & close the PR after reading comments.
Good to see all of you contributing to this PR. Please keep up the good job!

**Comment by gabrielsim** [comment]
Thanks for the input!

**Pull request closed without merging**